WcaOnRails/app/helpers/server_status_helper.rb

```diff
@@ -1,28 +1,14 @@
 # frozen_string_literal: true
 module ServerStatusHelper
-  def css_classes(missing_keys_empty, outdated_keys_empty)
-    if missing_keys_empty && outdated_keys_empty
-      class_heading = ""
-      class_panel = class_missing = class_outdated = "success"
-      title = "All good, the translation is up to date!"
-    elsif missing_keys_empty && !outdated_keys_empty
-      class_heading = "heading-as-link"
-      class_missing = "success"
-      class_panel = class_outdated = "warning"
-      title = "Some items in the translation are outdated"
-    elsif outdated_keys_empty && !missing_keys_empty
-      class_heading = "heading-as-link"
-      class_panel = "warning"
-      class_missing = "danger"
-      class_outdated = "success"
-      title = "Some items in the translation are missing"
-    else
-      # The only other choice is that both lists are not empty
-      class_heading = "heading-as-link"
-      class_panel = class_outdated = "warning"
-      class_missing = "danger"
-      title = "Some items in the translation are missing and some others are outdated"
-    end
-    [class_heading, class_panel, class_missing, class_outdated, title]
+  def css_classes(missing_keys_empty, unused_keys_empty, outdated_keys_empty)
+    class_missing = missing_keys_empty ? "success" : "danger"
+    class_unused = unused_keys_empty ? "success" : "warning"
+    class_outdated = outdated_keys_empty ? "success" : "warning"
+
+    all_good = missing_keys_empty && unused_keys_empty && outdated_keys_empty
+    class_panel = all_good ? "success" : "warning"
+    title = all_good ? "All good" : "Needs attention"
```

Comment:
With a 3rd boolean added to this method, we end up with `2**3 = 8` possible states for the translation. I didn't feel like writing something clever to turn this into a sentence, so I just went with "Needs attention". @viroulep, you went to all the effort of writing the previous sentences, are you okay with this simplification?